lua/wikis/marvelrivals/Infobox/Character/Custom.lua

```diff
@@ -51,13 +86,27 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+---@return Widget[]
+function CustomHero:_getRole(args)
+    local role = (args.role or ''):lower()
+    local roleLookup = {
+        duelist = { DUELIST },
+        strategist = { STRATEGIST },
+        vanguard = { VANGUARD }
+    }
+
+    return roleLookup[role] or {}
+end
+
 ---@param lpdbData table
 ---@param args table
 function CustomHero:addToLpdb(lpdbData, args)
 	lpdbData.information = args.name
```

Comment:
Is this needed? The data is is already stored in the name field in lpdb?

Reply:
Ahh, fair point. Ill repurpose this then